src/Task11/task11.php

```diff
@@ -0,0 +1,33 @@
+<?php
+function newElement(string $oldLast, string $last) : string
+{
+    $n = strlen($oldLast);
+    $tempStr = "";
+    $delta = 0;
+    for ($i = 0; $i < $n; $i++) {
+        $sum = (int) $oldLast[$i] + (int) $last[$i] + $delta;
+        $delta = intdiv($sum, 10);
+        $tempStr .= (string) $sum % 10;
+    }
+    if (strlen($last) > $n) {
+        $delta += (int) $last[$i];
+    }
+    if ($delta) {
+        $tempStr .= (string) $delta;
+    }
+
+    return $tempStr;
+}
+
+$element1 = '0';
```

Comment:
Why this code is out of the function? Write single function and then call it.

Reply:
there are 3 lines outside the function.
Why drive them into a function.
Then there will be 2 functions.